Enterprise-CMCS/macpro-mdct-mfp
# [MFP] SAR - R,E,&T - Quarter and Repeating Fields Logic

### Description



### Related ticket(s)


CMDCT-3021

---
### How to test


- Login, create a work plan, and (optionally) add other target populations under transition benchmarks 
- Leave form, add a new SAR 
- Navigate to "Number of MFP transitions in the reporting period" under "Recruitment, Enrollment, and Transitions" 
- Confirm that you see the correct reporting period and corresponding quarters within that period, and all the target populations per quarter ([Figma here](https://www.figma.com/file/oOJMUZerbafZefLQQ33GwI/MFP_Playground?type=design&node-id=246-53705&mode=design&t=acyOhY5aHn9Qfi7R-0))
- Go to other pages under RET and confirm that they are unchanged

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
